src/Featurizers/Components/Details/EstimatorTraits.h

```diff
@@ -15,7 +15,7 @@ namespace Details {
 ///  \brief         Base declaration to determine if a type has the method
 ///                 'has_created_transformer'.
 ///
-template <typename, typename T>
+template <typename T, typename FunctionT>
 struct HasHasCreatedTransformerMethodImpl {
     static_assert(std::integral_constant<T, false>::value, "Second template parameter must be a function type");
 };

```

Comment:
Don't you also need to switch the `T` in the static assert too?

Reply:
Either one will work in this case, as it is just needs to be a template so that it will cause a compiler error but only be invoked if the specialization is selected for compilation.